front/src/common/Map/MapKey.js

```diff
@@ -11,7 +11,12 @@ export default function MapSettings(props) {
   const { t } = useTranslation(['translation', 'map-key']);
   return (
     <div className={`map-modal map-modal-dark ${active ? 'active' : ''}`}>
-      <div className="h2 text-light">{t('map-key:keyTitle')}</div>
+      <div className="d-flex justify-content-between align-items-start">
+        <div className="h2 text-light">{t('map-key:keyTitle')}</div>
+        <button type="button" className="close text-light" onClick={toggleMapKey}>
+          ×
```

Comment:
Use rather `&times;` instead of the x
```suggestion
          &times;
```

Reply:
Done